sdk/diffgram/schema/attribute.py

```diff
@@ -0,0 +1,63 @@
+class Attribute:
```

Comment:
Great that you added this classes! I think we'll probably need to notate on this release that there is a breaking change for when getting the schema and the attributes. I think it might be a good idea to explain the old way vs new way of getting things. This is just for endpoints that already exists. 

The rest of the code looks great to me! Thanks a lot for adding this to the SDK VitaliI!

Reply:
@PJEstrada actually there are no breaking changes, I did not delete anything. We should add this to notes for sure, and mention that old way to create attributes will be deprecated soon